src/screens/Feed/components/Comment/component.js

```diff
@@ -142,7 +142,7 @@ export class Comment extends PureComponent {
         fields: {
           users: 'slug,avatar,name',
         },
-        include: 'user',
+        include: 'user,uploads',
```

Comment:
Should we take this out as we aren't supporting comment uploads at this point?

Reply:
I think it should be fine even if it's left in since it will handle the display of the uploads fine. It's also future-proofing a bit as when the time comes that we do support uploads in comments, we don't have to push an update so users can view them.